app/views/users/show.html.erb

```diff
@@ -5,5 +5,47 @@
 </div>
 <br>
 <div class='viewing-parties'>
+  <% @user.hosted_parties.each do |party| %>
+    <% movie = MovieFacade.new(party.movie_id).movie_details_show %>
```

Comment:
definitely unsure as to whether we should use this in the show view but don't know how else to iterate through all the movies that a user is hosting or invited to...